src/main/java/onlinebookstoreapp/model/Book.java

```diff
@@ -0,0 +1,27 @@
+package onlinebookstoreapp.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity(name = "books")
+@Data
```

Comment:
Forget abuut @Data for entities, @Data consists of @EqualsAndHashCode and @ToString, those two may lead to Lazy loading exception. Use @Getter and @Setter instead